wdl/ExpansionHunter.wdl

```diff
@@ -142,3 +165,145 @@ task RunExpansionHunter {
         maxRetries: runtime_attr.max_retries
     }
 }
+
+task SplitVariantCatalog {
+    input {
+        File variant_catalog
+        Int batch_size
+        String? split_variant_catalog_docker
+        String? output_prefix
+        RuntimeAttr? runtime_attr_override
+    }
+
+    output {
+        Array[File] catalogs_json = glob("${output_prefix_}*.json")
```

Comment:
One tricky thing about globs is maintaining order. If your outputs are `output_prefix_0.json`, `output_prefix_1.json`, `output_prefix_2.json`, etc. then `glob` will sort lexicographically and could mess up the order e.g. if you have more than 10 shards: `output_prefix_0.json`, `output_prefix_1.json`, `output_prefix_10.json`, `output_prefix_11.json`, etc. This could cause problems for your concatenation later.

The workaround we usually do is to pad the index with 0's, e.g. `output_prefix_000.json`, `output_prefix_001.json`, `output_prefix_002.json` etc. In theory you could figure out the number of necessary padding digits by counting the number of entries before doing the splitting. To me that seems a little complicated - I think you can just pad with 6 digits and call it a day.

Reply:
Good point, thanks! I added padding to the counter.

I'm actually not sure about the necessity of `output_prefix` for this task. It results in filenames such as `variant_catalog.jsonvariant_catalog_subset_000000.json`. Do you think it would be safe to remove `output_prefix`?